webapp/components/survey/CategoryDetails/ItemDetails/index.js

```diff
@@ -0,0 +1 @@
+export { default } from '../LevelDetails/ItemDetails'
```

Comment:
doesn't this folder must be moved as well?

Reply:
yes, I did something wrong while moving the files around...